src/Prelude/Linear.hs

```diff
@@ -71,6 +72,12 @@ maybe :: b -> (a ->. b) -> Maybe a ->. b
 maybe x _ Nothing = x
 maybe _ f (Just y) = f y
 
+-- XXX: temporary
```

Comment:
Ok. It's temporary. But what will obsolete it?

Reply:
Multiplicity polymorphism. This function is useful only because we don't have enough polymorphism to give a linear function to one expecting a non-linear arrow, so it's a temporary fix to make that work.
I was thinking of changing its name to `eta` since it has the same effect as eta-expanding `f` at use-sites, if you think that would be more useful.

Reply:
I've added a comment to reflect this.